docs/source/conf.py

```diff
@@ -292,5 +296,57 @@ def inject_minigalleries(app, what, name, obj, options, lines):
         lines.append("\n")
 
 
+def inject_weight_metadata(app, what, name, obj, options, lines):
+
+    if obj.__name__.endswith("_Weights"):
+        lines[:] = ["The model builder above accepts the following values as the ``weights`` parameter:"]
+        lines.append("")
+        for field in obj:
+            lines += [f"**{str(field)}**:", ""]
+
+            table = []
+            for k, v in field.meta.items():
+                if k == "categories":
+                    continue
+                elif k == "recipe":
+                    v = f"`link <{v}>`__"
+                table.append((str(k), str(v)))
+            table = tabulate(table, tablefmt="rst")
+            lines += [".. table::", ""]
+            lines += textwrap.indent(table, " " * 4).split("\n")
+            lines.append("")
+
+
+def generate_classification_table():
+
+    weight_enums = [getattr(M, name) for name in dir(M) if name.endswith("Weights")]
+    weights = [w for weight_enum in weight_enums for w in weight_enum]
+
+    column_names = ("**Weight**", "**Acc@1**", "**Acc@5**", "**Params**", "**Recipe**")
+    content = [
+        (
+            f":class:`{w} <{type(w).__name__}>`",
```

Comment:
Shouldn't this make the enum values are not hyperlinks on the HTML page? They don't seem to be [here](https://output.circle-artifacts.com/output/job/aaddb6cd-567c-4552-90fa-296878ff5ab1/artifacts/0/docs/models.html#table-of-all-available-classificaiton-weights).

Reply:
They are link, but only for those pages that exist. Ultimately this will be all of them, but right now this only works for Resnet and VGG weights. Try e.g. ResNet101_Weights.IMAGENET1K_V2